includes/definitions/airos-af.yaml

```diff
@@ -1,6 +1,6 @@
 os: airos-af
 text: 'Ubiquiti AirFiber'
-type: network
+type: wireless
```

Comment:
This will need a notification to be sent out for this change. https://github.com/librenms/librenms.github.io/blob/master/notifications.rss - See the last commit and copy this. Do 4 days ahead of when you create the notifications.rss PR